.github/workflows/spam-comment-detection.yaml

```diff
@@ -0,0 +1,45 @@
+name: Spam detection
+
+on:
+  issue_comment:
+    types: [created, edited]
+
+jobs:
+  detect-spam:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Node.js
+        uses: actions/setup-node@main
+
+      - name: Check for Spam
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const comment = process.env.COMMENT_BODY
+            const spam_words = ['Neuron Support Request']
+            const comment_id = process.env.COMMENT_ID
+            const issue_number = process.env.ISSUE_NUMBER
+            const owner = process.env.REPO_OWNER
+            const repo = process.env.REPO_NAME
+            const EXTERNAL_LINK_REGEXT = /https:\/\/(?!((\w+\.)?github\.com|github\.com|(\w+\.)?magickbase\.com|(\w+\.)?nervos\.org))/gi
+            if (spam_words.some(w => comment.includes(w))) {
+              console.info(`Spam comment: ${comment}`)
+              github.rest.issues.deleteComment({ owner, repo, comment_id })
+            } else if (EXTERNAL_LINK_REGEXT.test(comment)) {
```

Comment:
Should we skip the verification when the creator is a collaborator?

Reply:
I think this annotation should be for all links and users should be cautious even if the message is from a collaborator.